lib/Conversion/ImportVerilog/Structure.cpp

```diff
@@ -0,0 +1,158 @@
+//===- Structure.cpp - Slang hierarchy conversion -------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "ImportVerilogInternals.h"
+#include "slang/ast/Compilation.h"
+
+using namespace circt;
+using namespace ImportVerilog;
+
+//===----------------------------------------------------------------------===//
+// Module Member Conversion
+//===----------------------------------------------------------------------===//
+
+namespace {
+struct MemberVisitor {
+  Context &context;
+  Location loc;
+  OpBuilder &builder;
```

Comment:
It's just for convenience to have a `builder` available in the visit functions. It's an "alias" for `context.builder` or having to have a `getBuilder()` method :grimacing:

Reply:
Having a constructor that assigns that alias from the given context would force this invariant and make it more obvious when reading the code. It would also make it even more convenient to use the visitor 😉

Reply:
Haha good point. Hold my beer.